custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -348,22 +302,18 @@ def transform(self, model=None, **kwargs):
         -------
         pd.DataFrame
         """
-        input_filename = kwargs.get(StructuredDtoKeys.FILENAME)
         input_binary_data = kwargs.get(StructuredDtoKeys.BINARY_DATA)
         if self._custom_hooks.get(CustomHooks.READ_INPUT_DATA):
             try:
-                data = self._custom_hooks[CustomHooks.READ_INPUT_DATA](
-                    input_filename if input_filename is not None else input_binary_data
-                )
+                data = self._custom_hooks[CustomHooks.READ_INPUT_DATA](input_binary_data)
```

Comment:
Not fully aware for the context, but can you guarantee that the `self._custom_hooks` contains that entry?

Reply:
We check if before: ` if self._custom_hooks.get(CustomHooks.READ_INPUT_DATA):`